applications/DEMApplication/custom_strategies/strategies/explicit_solver_strategy.cpp

```diff
@@ -1794,8 +1790,8 @@ namespace Kratos {
         total_inelastic_frictional_energy = 0.0;
         double& total_inelastic_viscodamping_energy = r_process_info[PARTICLE_INELASTIC_VISCODAMPING_ENERGY];
```

Comment:
It's not to be fixed in this PR, but I consider using the ProcessInfo a bit ugly here. It is not meant to save physical quantities, but variables related to the process. We need to think a different place to put this information...

Reply:
I agree.
However, the ProcessInfo is never used to save or retrieve the energy values, only in this function (ExplicitSolverStrategy::CleanEnergies) that it is setting the energies to zero.
This is why I removed the call to this function in the beginning of the time step, as it was useless.
I would go further and remove the function.